conans/test/unittests/client/tools/files/remove_files_by_mask_test.py

```diff
@@ -0,0 +1,53 @@
+import os
+import unittest
+
+from conans.client.tools.files import remove_files_by_mask, chdir
+from conans.test.utils.test_files import temp_folder
+from conans.test.utils.tools import TestClient
+
+
+class RemoveFilesByMaskTest(unittest.TestCase):
+    def remove_files_by_mask_test(self):
+        tmpdir = temp_folder()
+        client = TestClient(current_folder=tmpdir)
```

Comment:
I think `client.current_folder` is already a temp folder

Reply:
changed to use `client.current_folder`